### DISCUSSION TITLE
How to use jtxBoard in desktop (Win) environment

### DISCUSSION THREAD

**Discussion by greg4b** [TechbeeAT/jtxBoard#676] [General]
Hello,

I'm wondering if this app / programm can be used on a win10 desktop machine without administrator privileges?

Thanks & Greetings from Graz,

Greg

**Comment by patrickunterwegs** [comment]
Hey @greg4b !
Basically on Windows 11 you could download the Android Subsystem for Windows together with the Amazon appstore. From there you could download DAVx5 and jtx Board. 
However, for Windows 10 I am not aware of an easy way to get the applications installed. I have read that there are ways to get a Android Subsystem for Windows also running on Windows 10, but I've never tried it myself. But I'm pretty sure you'd need admin rights...

Regrettably there are still no other Windows applications that would support journals, notes and tasks. 
For tasks only you could try a portable version of Thunderbird!
